geomesa-hbase/geomesa-hbase-tools/bin/aws-utils/.aws-bootstrap-actions.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+# Do not run this script manually. It is part of the automated bootstrap process and is not written for end user use.
```

Comment:
i'd probably not make it a hidden file though unless there is a compelling reason

Reply:
It is a hidden file :)

Reply:
Just to keep people from running it. They don't even need to know it exists.